test/middleware.js

```diff
@@ -124,3 +126,102 @@ describe('Using middleware', function () {
   });
 
 });
+
+describe('Using middleware to compile .sass', function () {
+  var server = connect()
+    .use(middleware({
+      src: __dirname,
+      dest: __dirname,
+      indentedSyntax: true
+    }));
+
+  beforeEach(function (done) {
+    fs.exists(cssfile, function (exists) {
+      if (exists) {
+        fs.unlink(cssfile, done);
+      } else {
+        done();
+      }
+    });
+  });
+
+  describe('successful file request', function () {
+
+    it('serves a file with 200 Content-Type css', function (done) {
+      request(server)
+        .get('/test.css')
+        .set('Accept', 'text/css')
+        .expect('Content-Type', /css/)
+        .expect(200, done);
+    });
+
+    it('serves the compiled contents of the relative sass file', function (done) {
+      var filesrc = fs.readFileSync(sassfile),
+          result = sass.renderSync({ data: filesrc.toString(), indentedSyntax: true });
+      request(server)
+        .get('/test.css')
+        .expect(result.css)
+        .expect(200, done);
+    });
+
+    it('writes the file contents out to the expected file', function (done) {
+      var filesrc = fs.readFileSync(sassfile),
```

Comment:
This test is failing on TravisCI.
Is it passing locally, when running `npm test`?

If so, please consider adding `this.timeout = 5000;` before line 168,